docs/site/Getting-started.md

```diff
@@ -42,13 +42,13 @@ Answer the prompts as follows:
 ? Project root directory: (getting-started)
 ? Application class name: StarterApplication
 ? Select features to enable in the project:
-❯◉ Enable tslint
- ◉ Enable prettier
- ◉ Enable mocha
- ◉ Enable loopbackBuild
- ◉ Enable vscode
- ◉ Enable repositories
- ◉ Enable services
+❯◉ Enable tslint: adds a linter with pre-configured lint rules
+ ◉ Enable prettier: installs prettier for consistent code formatting
```

Comment:
I think this is not only installing prettier, but also adding new npm scripts and hooking them into `npm test` and `npm lint`.

Can we explain the feature at high-level, maybe explain why users may want to use this feature. Don't explain implementation details.